### PR TITLE
Update config files to be in sync with their counterparts in kube repo.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
@cadvisorJenkinsBot: retest this please

File: deploy/kube-config/google/heapster-controller.json [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
 	    "spec": {
 		"containers": [
 		    {
-			"name": "heapster",
 			"image": "kubernetes/heapster:v0.17.0",
+			"name": "heapster",
 			"command": [
 			    "/heapster",
-			    "--source=kubernetes:''",
+			    "--source=kubernetes:''"
```
Comment (mvdan) [inline deploy/kube-config/google/heapster-controller.json]: typo?

File: deploy/kube-config/influxdb/heapster-controller.json [hunk 2]
```diff
@@ -25,7 +30,7 @@
 			"name": "heapster",
 			"command": [
 			    "/heapster",
-			    "--source=kubernetes:https://kubernetes",
+			    "--source=kubernetes:''"
```
Comment (mvdan) [inline deploy/kube-config/influxdb/heapster-controller.json]: another typo?

File: deploy/kube-config/google/heapster-controller.json (hunk 1 above)
Comment (vishh) [inline deploy/kube-config/google/heapster-controller.json]: Good catch @mvdan

File: deploy/kube-config/influxdb/heapster-controller.json (hunk 2 above)
Comment (vishh) [inline deploy/kube-config/influxdb/heapster-controller.json]: Good eyes again. Fixed. PTAL

**Comment by mvdan** [comment]
LGTM

**Pull request merged**